sources/models/currentStory.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+var mongoose = require('mongoose');
+var currentStorySchema = new mongoose.Schema({
+	storyAsset:String,
+	authorName: String, 
+	authorLogo: String,
```

Comment:
Make these as required: true

Reply:
We will work on input validation on all the schemas